gaboon/config.py

```diff
@@ -212,41 +224,53 @@ def write_dependencies(self, new_dependencies: list):
 
         self.dependencies = new_dependencies
 
+    def get_base_dependencies_install_path(self) -> Path:
+        project_root = self._project_root
+        base_install_path = project_root / self.project.get(
+            DEPENDENCIES_FOLDER, DEPENDENCIES_FOLDER
```

Comment:
hmm, kind of a strange thing to default to

Reply:
why is that?

Reply:
it means it can be either a key or a string right? why not just pick one, too dynamic is also confusing